x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -167,16 +168,21 @@ export const isolationRequestHandler = function (
       commentLines.push(`\n\nWith Comment:\n> ${req.body.comment}`);
     }
 
-    caseIDs.forEach(async (caseId) => {
-      (await endpointContext.service.getCasesClient(req)).attachments.add({
-        caseId,
-        comment: {
-          comment: commentLines.join('\n'),
-          type: CommentType.user,
-          owner: APP_ID,
-        },
-      });
-    });
+    // Update all cases with a comment
+    if (caseIDs.length > 0) {
```

Comment:
do we need this check?  `map` on an empty array would just do nothing, right?

Reply:
I was trying to avoid the additional `await` if not needed. yeah, its minimal since the promise would be resolved on the next `tick`. I can remove it if you prefer to not have it

Reply:
it's optional IMO - it would also have the benefit of remove the es-lint ignore above this.  However, this file will change again before we ship with potentially more complexity, so perhaps we'll need to break some things up anyways.